### DISCUSSION TITLE
Unable to auto-login user after email confirmation using supabase.auth.resend method

### DISCUSSION THREAD

**Discussion by Elvincth** [supabase/supabase#15583] [Questions]
I'm currently using the supabase.auth.resend method in Supabase to resend email confirmation to users who have signed up but not yet confirmed their email address. However, unlike the supabase.auth.signUp method, the supabase.auth.resend method does not support the emailRedirectTo option, which would enable a seamless redirect after email confirmation.

As a result, I'm unable to automatically log in the user after they confirm their email address, which can result in a disjointed authentication flow and potentially cause users to drop off. Can the Supabase team suggest a workaround or alternative method to achieve auto-login after email confirmation using the supabase.auth.resend method?

```js
    const { error } = await supabase.auth.signUp({
      email,
      password,
      options: {
        emailRedirectTo: `${process.env.NEXT_PUBLIC_SITE_URL}/api/auth/callback`,
      },
    });
```

Not support in `supabase.auth.resend` method:
```js
 const { error } = await supabase.auth.resend({
      type: "signup",
      email,
      options:{
        emailRedirectTo: "..." //NOT SUPPORTED YET
    }
```

**Comment by GaryAustin1** [comment]
https://github.com/supabase/gotrue/issues/1142
Open issue on it.
